include/libtorrent/create_torrent.hpp

```diff
@@ -366,8 +366,7 @@ namespace libtorrent {
 		bool m_include_symlinks:1;
 	};
 
-namespace detail {
-
+	namespace detail {
```

Comment:
actually, `inline void nop(piece_index_t) {}` is already double indented, do you really prefer remove one indentation level here?